client/src/views/Game.vue

```diff
@@ -154,13 +168,24 @@
               </v-row>
             </v-card>
 
-            <Card
-              v-for="(card, i) in gameState.player_cards"
-              :key="i"
-              :number="card.value"
-              :color="card.color"
-              @click.native=" (card.value == 'W' || card.value == 'W4') ? selectWildColor(card) : playCard(card)"
-            ></Card>
+            <div id=myCards>
```

Comment:
Can we remove the `id` from the div? So far as I know, it's best practice to avoid using them as it is generally too difficult to ensure uniqueness on the final rendered page.

Reply:
Mmmmaybe? The problem is that in order to navigate between cards using the keyboard we currently need to be able to unambiguously select the group of cards in our hand so that I can use the array positions of the data structure to figure out what the “next” card is. Because we use card classes in other areas (discard pile) I can’t just take the nth Card on the page, and I’m not familiar with any ways of getting the nth instance of a class in JavaScript. (CSS selectors maybe?)

I could use a much longer ID value, but I couldn’t think of any way to get rid of the ID entirely. If someone has a solution to that I could happily implement it.

Reply:
I think the proper solution to this, at least with Vue is to use refs to get specific element. The Game page now uses refs to get individual cards, so you could look at it on dev to see how that works. You can basically get them with an index.

Reply:
I've switched over to `$refs` but unfortunately dev is broken at the moment and I can't test my changes until that gets fixed. I'll look into fixing it, but until I can test the new changes I'm not going to push them to this review unless asked.

Reply:
Okay, I think this is fixed. I played a few games with the new controls and it's a little clunky, but I think it's as good as it'll get with the amount of churn we have in this repo.